packages/packages_linux.go

```diff
@@ -127,6 +127,9 @@ func GetInstalledPackages(ctx context.Context) (Packages, error) {
 			pkgs.Deb = deb
 		}
 	}
+	if cos, err := InstalledCOSPackages(); err == nil {
```

Comment:
Your missing the check for COSPkgInfoExists

Reply:
Fixed in the new commit.

By the way, the previous commit I made seems lost. How can I preserve the patch history, like keeping the past "patch sets" for a specific CL in Gerrit?